pkg/skaffold/schema/latest/v2/config.go

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2
+
+import "github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/util"
+
+const Version string = "skaffold/v3alpha1"
+
+// NewSkaffoldConfig creates a SkaffoldConfig
+func NewSkaffoldConfig() util.VersionedConfig {
+	return new(SkaffoldConfig)
+}
+
+func (c *SkaffoldConfig) GetVersion() string {
+	return c.APIVersion
+}
+
+// SkaffoldConfig holds the fields parsed from the Skaffold configuration file (skaffold.yaml).
+type SkaffoldConfig struct {
+	// APIVersion is the version of the configuration.
+	APIVersion string `yaml:"apiVersion" yamltags:"required"`
+
+	// Kind is always `Config`. Defaults to `Config`.
+	Kind string `yaml:"kind" yamltags:"required"`
+
+	// Metadata holds additional information about the config.
+	Metadata Metadata `yaml:"metadata,omitempty"`
+
+	// Dependencies describes a list of other required configs for the current config.
+	Dependencies []latest_v1.ConfigDependency `yaml:"requires,omitempty"`
+
+	// Pipeline defines the Build/Test/Deploy phases.
+	Pipeline `yaml:",inline"`
+
+	// Profiles *beta* can override be used to `build`, `test` or `deploy` configuration.
+	Profiles []Profile `yaml:"profiles,omitempty"`
+}
+
+// Metadata holds an optional name of the project.
+type Metadata struct {
+	// Name is an identifier for the project.
+	Name string `yaml:"name,omitempty"`
+}
+
+// Pipeline describes a Skaffold pipeline.
+type Pipeline struct {
+	// Build describes how images are built.
+	Build latest_v1.BuildConfig `yaml:"build,omitempty"`
+
+	// Test describes how images are tested.
+	Test []*latest_v1.TestCase `yaml:"test,omitempty"`
+
+	// Render describes how the original manifests are hydrated, validated and transformed.
+	Render RenderConfig `yaml:"render,omitempty"`
```

Comment:
Sorry, just saw the config in detail now. 
Shd we instead use ?
```
manifests:
   
    validate:

    transform:

```

Reply:
Yeah, I used the field name from the design doc. Do we get a conclusion about the render UX? I can renaming these fields once we determin the final naming (and update the design doc). For now, this is just to align with the design doc. Whom do you suggest determining (~~documenting~~) the render field names from the survey?

@tejal29 @viglesiasce

Reply:
I updated the UX from "render" to "manifests" and keep the struct name as "render" (manifests seem to be over-used). I can rename the fields if there're some better choices coming up.